src/markdown/mas/steekproefontwerp_zandleemstreek.Rmd

```diff
@@ -376,6 +389,37 @@ mapview(st_crop(perimeters_data, detail), color = "red", alpha.regions = 0,
   mapview(st_crop(exclusie_punten, detail), col.regions = "black", legend = FALSE)
 ```
 
+### Exclusie op basis van zichtbaarheid
+
+Berekening cumulatieve zichtbaarheidsindex binnen straal van 25 m rond telpunt (zone bewegingsvrijheid van de waarnemer) in een raster (resolutie 5 m) van locaties die 10 m van elkaar liggen.
+Door middel van Digital Surface Model (DSM) en Digital Terrain Model (DTM).
+Aanpassing aan **GVI** package.
+Zie: `visibility.Rmd`.
+
+In de pipeline werd gebruik gemaakt van batching per 200 telpunten om rekentijd in te perken.
+Zie:\
+<https://docs.ropensci.org/tarchetypes/reference/tar_group_size.html>\
+<https://books.ropensci.org/targets/dynamic.html#performance-and-batching>
+
+We verwijderen telpunten waar de cumulatieve 'viewshed of visibility' (cvvi) kleiner is dan 0.10.
```

Comment:
```suggestion
We verwijderen telpunten waar de cumulatieve 'viewshed visibility index' (cvvi) kleiner is dan 0.10.
```
Eventueel eens in aparte Rmd een aantal voorbeeldkaartjes (orthofoto achtergrond) maken waarin de cumulatieve viewshed gevisualiseerd wordt (via output_type = "summed viewshed") voor gevallen waar de cvvi ongeveer gelijk is aan 0.05, 0.1, 0.15, 0.2, 0.3, ...?